test/seneca.test.js

```diff
@@ -1170,6 +1170,7 @@ describe('seneca', function () {
     si.options({errhandler: done})
 
     si.use('entity')
+    si.use('mem-store')
 
```

Comment:
@mcdonnelldean don't forget about the mem store enabled by default. You are right that would be a kind of (small) breaking change